src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, { useEffect, useMemo, useState } from 'react';
+import { UserWarning } from './components/UserWarning';
+import { TodoInput } from './components/TodoInput';
+import { TodosList } from './components/TodosList';
+import { Todo } from './types/Todo';
+import { TodosFilter } from './components/TodosFilter';
+import { NotificationError } from './components/NotificationError';
+import { Filter } from './types/Filter';
+import { filterTodos } from './utils/helpers';
+import { getTodos } from './api/todos';
+import { ErrorAction } from './types/ErrorAction';
+
+const USER_ID = 6796;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
```

Comment:
this state can be removed and you can use `useMemo` hook to calculate visible todos

Reply:
still not fixed